app/src/main/java/com/example/myapplication/ui/MainActivityScreen.kt

```diff
@@ -25,25 +25,32 @@ import kotlin.random.Random.Default.nextInt
 fun DefaultPreview() {
     val colors = listOf(Gray, Red, Green, Yellow, Magenta)
     val items = mutableListOf<ItemViewState>()
-    for (i in 0..21) {
+    for (i in 1..4) {
         val colorId = nextInt(0, 5)
-        items.add(ItemViewState("title $i", background = colors.get(colorId)))
+        items.add(ItemViewState("title $i", background = colors.get(colorId), widthRatio = nextInt(1, 3).toFloat()))
     }
 
     val layoutViewState =
         LayoutViewState(items, columnsCount = 4, layoutBackground = Color.Cyan, itemsHeight = ItemsHeight.Dynamic(20))
 
-    FlexibleGrid(
+//    FlexibleGrid(
+//        layoutViewState = layoutViewState,
+//        gridWidth = 300,
+//        paddingValues = PaddingValues(start = 15.dp, end = 15.dp, top = 15.dp, bottom = 15.dp),
+//        verticalArrangement = Arrangement.spacedBy(15.dp),
+//        horizontalArrangement = Arrangement.spacedBy(15.dp)
+//    )
+//
+//    HorizontalScroll(
+//        layoutViewState = layoutViewState,
+//        layoutWidth = 300,
+//        paddingValues = PaddingValues(start = 15.dp, end = 15.dp, top = 15.dp, bottom = 15.dp),
+//        horizontalArrangement = Arrangement.spacedBy(15.dp)
+//    )
```

Comment:
Is it needed here?

Reply:
This comments needed for other view examples. It will help us to usage and transfer this code to another project